src/modules/node/start_chroot_script

```diff
@@ -17,3 +17,6 @@ curl -fsSL https://deb.nodesource.com/setup_16.x | bash - &&\
 curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | tee /usr/share/keyrings/yarnkey.gpg >/dev/null
 echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | tee /etc/apt/sources.list.d/yarn.list
 apt update && apt install nodejs yarn -y 
+
+#debug - trying this to get around network errors?
+yarn config set network-timeout 1200000
```

Comment:
Remove this, should not be necessary. If it is we need to figure out what's going on.